### DISCUSSION TITLE
When will added the possibility of using both LTR and RTL in same page ?

### DISCUSSION THREAD

**Discussion by devhoussam** [twbs/bootstrap#33488] [Q&A] [answered]
I'm using laravel framework as my Backend Framework and sometimes I work on projects for Arabic clients I want to know when will added the possibility of using both LTR and RTL in same page in BS V5 ??

**Comment by ffoodd** [comment, marked as answer]
We added a workaround for this in out RTL docs, mostly based on RTLCSS to wrap our CSS with something like `.rtl` and `.ltr`.

You'd still need to separate CSS files but it'd allow you to use both at the same time, and only require you put those classes appropriately to apply one or another CSS.

**Comment by motahharm** [comment]
Hi @devhoussam 

you can have two css files.
rtl one of them with RTLcss

**Comment by WinterSilence** [comment]
[read manual](https://getbootstrap.com/docs/5.1/getting-started/rtl/#ltr-and-rtl-at-the-same-time)
